temba/msgs/models.py

```diff
@@ -1212,7 +1206,7 @@ def create_incoming(cls, channel, urn, text, user=None, date=None, org=None, con
             analytics.gauge('temba.msg_incoming_%s' % channel.channel_type.lower())
 
         # ivr messages are handled in handle_call
-        if status in (PENDING, INTERRUPTED) and msg_type != IVR:
+        if status == PENDING and msg_type != IVR:
```

Comment:
no that should be alright for now, the USSD async handlers are basically just calling this and let it be handled the same way it's done with sms (since there's no difference from that point whatsoever). The difference comes from the APIs (like InfoBip) that need sync commnucation (handle the message right away and generate the reply in the response to the API request). I could possibly refactor it a bit to use its own handler only, but for the sake of VUMI there's no difference between handling an sms and a ussd msg now.